librad/src/git/storage.rs

```diff
@@ -991,11 +991,11 @@ mod tests {
         user.sign_owned(&user_key).unwrap();
         let verified_user = user
             .clone()
-            .check_signatures()
+            .verify_signatures()
             .unwrap()
-            .check_signatures_ownership(&cache)
+            .verify_certifiers(&cache)
             .unwrap()
-            .check_update(&None)
+            .verify_update(&None)
```

Comment:
Can this just be `user.verify(&cache)` now?

Reply:
Right!